Resources/Textures/_RMC14/Objects/Misc/target.yml

```
@@ -0,0 +1,83 @@
+- type: entity
+  name: human target
+  id: RMCTargetHuman
+  parent: BaseStructureDynamic
+  abstract: true
+  description: A shooting target. This one is a Human.
+  suffix: RMC
+  components:
+  - type: Sprite
+    sprite: Objects/Specific/Security/target.rsi
+    state: target_h
+    noRot: true
+  - type: Repairable
+  - type: DamagePopup
+    allowTypeChange: true
+    damagePopupType: Combined
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeCircle
+          radius: 0.35
+        density: 200
+        mask:
+        - FullTileMask
+        layer:
+        - MobLayer
+  - type: InteractionOutline
+  - type: Physics
+  - type: Damageable
+    damageContainer: Inorganic
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 100000000000000000000
+      behaviors:
+        - !type:DoActsBehavior
+          acts: [ "Destruction" ]
+    - trigger:
+        !type:DamageTrigger
+        damage: 500000000000000000000
+      behaviors:
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          CMSteel:
+            min: 5
+            max: 5
+      - !type:PlaySoundBehavior
+        sound:
+          collection: MetalBreak
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
+
+- type: entity
+  name: syndicate target
+  id: RMCTargetSyndicate
+  parent: RMCTargetHuman
+  description: A shooting target. This one is a syndicate agent.
+  components:
+  - type: Sprite
+    sprite: Objects/Specific/Security/target.rsi
+    state: target_s
+
+- type: entity
+  name: clown target
+  id: RMCTargetClown
+  parent: RMCTargetHuman
+  description: A shooting target. This one is a clown.
+  components:
+  - type: Sprite
+    sprite: Objects/Specific/Security/target.rsi
+    state: target_c
+
+- type: entity
+  name: felinid target
+  id: RMCTargetStrange
+  parent: RMCTargetHuman
+  description: A shooting target. This one is a Felinid.
```

I don't think it's meant to be literally a felinid despite the ears but either way since this one isn't used in 13 you can just copy the upstream description
```suggestion
  description: A shooting target. You aren't quite sure what this one is, but it seems to be extra robust.
```

---

Alrighty, just had it like that since we have felinids.